### ISSUE TITLE
[ADE Intake]: ADE - GitHub Integration - Slack Internal Channel Notifications (Trigger: New Intakes)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#102848]
> [!NOTE]
> **QUICK SUMMARY:** This ticket is to test the integration of Slack with GitHub in the accessibility-de-internal channel. 

## About Your Team

- **Team name**: ADE
- **OCTO product owner**: Martha Wilkes
- **Product manager**: Naomi Eke
- **Slack channel**:  [accessibility-de-internal](https://dsva.slack.com/archives/C08B367JLLT)
- **Dedicated accessibility specialist on your team** (if you have one):
- **Accessibility Champ** (if you have one): `ALL`

## About Your Work
> [!TIP]
> _Tell us briefly about your work and the outcomes you’re aiming to achieve._

### What's the nature of your initiative and desired outcomes?

- Add integration for GitHub to channel
- Remove default triggers: Enabled by default and can be disabled:
    - Keep: issues 
    - Remove: pulls, commits, releases, deployments
- Trigger notifications for any new tickets with the label `ADE Intake`

<img width="397" alt="Slack post automation from GitHub
APP  5:10 PM This channel accessibility-de-internal will receive notifications from department-of-veterans-affairs/va.gov-team for issues,+label:'ADE Intake'" src="https://github.com/user-attachments/assets/0b97bd59-1a58-4854-bbc4-2a8dcc819b51" />

  
## Collaborate with ADE
[**Review our menu of services**,](https://github.com/department-of-veterans-affairs/va.gov-team/tree/f12c0655a990044374088eed0d48208e2a120708/teams/CAIA/accessibility) and check all the types of collaboration you're requesting, below:

### How would you like to collaborate with ADE?
- [ ] Research support
- [ ] Accessibility test planning
- [ ] Support design discovery
- [ ] Review wireframes or mockups
- [ ] PDF audit
- [ ] Staging / CodePen / Codespaces / Production audit
- [ ] Health check audit
- [ ] Training and education
- [ ] Spot check, general guidance
- [x] **Something else:**  `insert type of collaboration`
- [ ] **Unsure?** _Check this box if you're not sure what kind of support you need, need multiple types of support, and/or want to meet with us. We'll schedule an introductory meeting to learn more about your team and project._

## Provide Your Supporting Artifacts 

> [!TIP]
> **Include artifacts like your product outline, user flows, mockups and prototypes, etc.** 
> - **Figma/Mural:** Please tell us which screens/board/section to look at so we don't leave comments on old stuff. 
> - **Staging Audits:** We will need staging user(s) and scenarios provided.

### Links to Artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. 

- [Slack Channel for accessibility-de-internal](https://dsva.slack.com/archives/C08B367JLLT)
- [Notification Automation Message 5:10 p.m. EST Feb. 10, 20205](https://dsva.slack.com/archives/C08B367JLLT/p1739225413836779)

## Your Timeframe

> [!IMPORTANT]
> - We work with the majority product teams on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.
> - If you ever have any questions about the status of your ticket(s) please reach out to us in the [accessibility-help](https://dsva.slack.com/archives/C8E985R32) Slack channel by tagging Naomi Eke or Lakwi Perera.
> - We will notify VFS Teams via the weekly Team of Teams slide, if our bandwidth is impacted by holidays, PTO, planned research, larger projects, etc.
> - Additionally, feel free to access our [capacity calendar](https://calendar.google.com/calendar/embed?height=600&wkst=2&ctz=America%2FNew_York&mode=WEEK&src=Y19mOTgyMzUwZmM3M2E4MTYyYTEwODhhYjE0ZDAwZGQzNzMzNWIyYjU1OWE2MGQ2YjY3ZWI4MDIxNDE2ZjgxYzJhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20&color=%23616161).

### Is this work tied to a specific deadline?
- [ ] Yes (Month/Day/Year)
- [x] No
- [ ] I’m not sure

### Ticket Number
NA

### Which phases of the collaboration cycle have you completed?
_Select all that apply._
- [ ] Design intent
- [ ] Midpoint review
- [ ] Staging
- [x] None. This initiative isn’t going through the collaboration cycle.

```[tasklist]
### Next Steps for Your Team
- [x] Save and submit this intake ticket.
- [x] Start a new thread, and post a link to this ticket in the [accessibility-de-internal](https://dsva.slack.com/archives/C08B367JLLT/p1739225413836779) Slack channel by tagging Naomi Eke or Lakwi Perera.
- [x] Please don't close this intake when work is complete - ADE will close and track.
```

```[tasklist]
## Ticket Updates (ADE Internal)
- [x] Receive new intake ticket
- [x] Respond to Slack message
- [ ] Reviewed by OCTO/Martha
- [x] Connect related tickets in this ticket
- [x] Create child tickets, as they apply to the work requested.
- [x] Create Quick Summary for top of ticket as a note.
- [x] _Label with `Originator/Team` (product team or stakeholder requesting support)_
- [x] _Label date in the `Open Date` field_
- [x] _Label with `Estimate` (level of effort expected for this ticket)_
- [x] _Add `Assignee(s)` name(s) to ticket_ (include Naomi)
- [x] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [x] _Select a `Priority Level`_
- [x] _Update date in `Last Checked` field_
- [x] _Label with `Actual` (level of effort it took to complete this ticket)_
- [x] _Update date in_ `Closed Date`
```

```[tasklist]
## Related Tickets
- [ ] #102851
- [ ] #ticketnumbergoeshere2
- [ ] #ticketnumbergoeshere3
```

**Comment by sara-amanda** [comment]
https://github.com/department-of-veterans-affairs/va.gov-team/issues/102851#issuecomment-2649407896
